tools/secret-rotation/Azure.Sdk.Tools.SecretRotation.Configuration/PlanConfiguration.cs

```diff
@@ -1,37 +1,55 @@
-﻿using System.Text.Json;
+using System.Text.Json;
 using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
 
 namespace Azure.Sdk.Tools.SecretRotation.Configuration;
 
 public class PlanConfiguration
 {
-    [JsonPropertyName("name")]
-    public string? Name { get; set; }
+    private static readonly JsonSerializerOptions jsonOptions = new()
+    {
+        AllowTrailingCommas  = true,
+        Converters =
+        {
+            new JsonStringEnumConverter()
+        },
+        PropertyNameCaseInsensitive = true,
+        PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+        ReadCommentHandling = JsonCommentHandling.Skip,
+    };
+
+    private static readonly Regex schemaPattern = new (@"https\://raw\.githubusercontent\.com/azure/azure-sdk-tools/(?<branch>.*?)/schemas/secretrotation/(?<version>.*?)/schema\.json", RegexOptions.IgnoreCase);
```

Comment:
The version regex segment matches optional/any number, but I think we would want to require it?

Reply:
True.   I'll update to `(?<branch>.+?)`  and `(?<version>.+?)`.   I don't want to bake semver checking into the regex match.   I'll leave it up to version comparison later in the file if we want to restrict to specific versions or branches.   For now, I just want to separate is-our-file from not-our-file.